src/Definition/Resolver/ResolverInterface.php

```diff
@@ -4,6 +4,10 @@
 
 namespace Overblog\GraphQLBundle\Definition\Resolver;
 
+/**
```

Comment:
also add a `trigger_error` call please

Reply:
How do I add `trigger_error` in an interface?

Reply:
You should just added before the class or the interface definition.
You can see examples here:
https://symfony.com/doc/current/contributing/code/conventions.html

Reply:
Done